src/utils/upVote.utils.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const ncGamesApi = axios.create({
+  baseURL: "https://nc-games-m3wy.onrender.com/api/",
+});
+
+function upVote(inc_votes, review_id) {
+  return ncGamesApi.patch(`reviews/${review_id}`, {inc_votes})
+  .then((response) => {
+    return response.data.review.votes
+  })
+  .catch((error) => {
```

Comment:
You already have a catch block for this which is working fine, this here isn't needed as your catch block on your back end _should_ be handling this 😉